mmv1/third_party/terraform/resources/resource_compute_shared_vpc_service_project.go.erb

```diff
@@ -121,6 +131,12 @@ func resourceComputeSharedVpcServiceProjectDelete(d *schema.ResourceData, meta i
 	config := meta.(*Config)
 	hostProject := d.Get("host_project").(string)
 	serviceProject := d.Get("service_project").(string)
+	
+	if d.Get("deletion_policy").(string) == "ABANDON" {
```

Comment:
It would be great to match the other ABANDON implementations - unless there's a particular concern about doing it this way?

```suggestion
	if deletionPolicy := d.Get("deletion_policy"); deletionPolicy == "ABANDON" {
```